docs/index.md

```diff
@@ -7,6 +7,7 @@
 ## Introduction
 * [Installation Using Docker](installation/installation-using-docker.md) and [Native Installation](installation/native-installation.md) 
 * [Installation Using Docker on Production Server](installation/installation-using-docker-on-production-server.md)
+* [Recommended Workflow for Development on Shopsys Framework](introduction/recommended-workflow-for-development.md)
```

Comment:
Also, I am thinking about the added value of the article. I know Miro wanted this...I like having it documented as well but I would probably change the title to "Shopsys Framework Development Workflow" - I am not sure whether we should **recommend** it generally. Please discuss it with @LukasHeinz during business validation. 

If you went this way, it would be probalby better to move the link a liitle bit lower in the "Introduction" section

Reply:
I have updated it by your suggestion